pkg/registry/client_test.go

```diff
@@ -167,6 +206,15 @@ func (suite *RegistryClientTestSuite) Test_5_RemoveChart() {
 	suite.Nil(err)
 }
 
+func (suite *RegistryClientTestSuite) Test_7_Logout() {
+	// TODO: re-enable once fixed in oras (no error thrown on linux)
+	// err := suite.RegistryClient.Logout("this-host-aint-real-son:5000")
+	// suite.NotNil(err, "error logging out of registry that has no entry")
```

Comment:
we should be able to re-enable this now. Ill make the change once we move to a 0.4.0

Reply:
done